pkg/cmd/update/password.go

```diff
@@ -0,0 +1,107 @@
+package update
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"net/http"
+
+	"github.com/litmuschaos/litmusctl/pkg/apis"
+	"github.com/litmuschaos/litmusctl/pkg/types"
+	"github.com/litmuschaos/litmusctl/pkg/utils"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+var PasswordCmd = &cobra.Command{
+	Use: "password",
+	Short: `Updates an account's password.
+		Examples(s)
+		#update a user's password
+		litmusctl update password
+		`,
+	Run: func(cmd *cobra.Command, args []string) {
+		var (
+			updatePasswordRequest types.UpdatePasswordInput
+		)
+
+		credentials, err := utils.GetCredentials(cmd)
+		if err != nil {
+			utils.PrintError(err)
```

Comment:
@Baalekshan please also check if on error we want to end the flow or continue with the next steps. If we want to end in case of error, you can add os.Exit(1) here and in other places